tox.ini

```diff
@@ -1,11 +1,48 @@
 # Note: Run "python bootstrap.py" before running Tox, to generate metadata.
-#
-# To run Tox against all supported Python interpreters, you can set:
-#
-# export TOXENV='py27,py3{3,4,5,6},pypy,pypy3'
+[tox]
+envlist = py27,
```

Comment:
My preference here is to run tox only for the main python version ('python') and leave the rest to be executed by CI runners. Even if we wish to change this expectation, we should do it in a separate pull request.

Reply:
OK, I'll revert this part.

In the past I've just had CI manually override the environments and the bare `tox` command just run on all supported environments present on the user's local machine, but that's just because that's how it was set up in the projects I've worked on.

Reply:
I would say it's more a best practice to set the skip missing interpreters but still run the most we can on the user machine, instead of forcing just whatever version the user has as ``python``.